### PR TITLE
Address review suggestions

### PR DESCRIPTION
**Pull request by MaryamZi**
## Purpose
$title.

### CODE REVIEW

File: service-orchestration/README.md [hunk 1]
```diff
@@ -2,82 +2,82 @@
...
+            "phone": "8770586755",
+            "email": "johndoe@gmail.com",
+            "cardNo": "7844481124110331"
+        },
+        "doctor": "thomas collins",
+        "hospital_id": "grandoaks",
+        "hospital": "grand oak community hospital",
+        "appointment_date": "2023-10-02"
+    }
+    ```
```
Comment (MaryamZi) [inline service-orchestration/README.md:30]: MD view looks OK.

**Review by dulajdilshan — DISMISSED** [review]

File: service-orchestration/README.md [hunk 2]
```diff
@@ -567,9 +567,9 @@ Running executable
 
 Let's test the use case by sending a request to the service.
 
-#### Start the back end service
+#### Start the backend service
 
-Download the JAR file for the backend service from [here](https://github.com/ballerina-guides/integration-tutorials/blob/main/backends/hospital-service/) and execute the following command to start the service:
+Download the JAR file for the [backend service](https://github.com/ballerina-guides/integration-tutorials/blob/main/backends/hospital-service/hospitalservice.jar) and execute the following command to start the service:
 
 ```bash
```
Comment (mindula) [inline service-orchestration/README.md:574]: Let's indent this

File: sending-a-message-to-a-service/README.md [hunk 3]
```diff
@@ -15,121 +15,121 @@ To implement this use case, you will develop a REST service with a single resour
...
-```
+    ```bash
+    $ bal new sending-a-message-to-a-service
+    ```
 
 2. Introduce the source code in files with the `.bal` extension (e.g., the `main.bal` file). 
 
 Import the 
 - `ballerina/http` module to develop the REST API and define the client that can be used to send requests to the backend service
 - `ballerina/log` module to log some information for each client request
```
Comment (dulajdilshan) [inline sending-a-message-to-a-service/README.md]: Shouldn't we intend this?

File: service-orchestration/README.md (hunk 2 above)
Comment (dulajdilshan) [reply service-orchestration/README.md:574]: This part doesn't belong to a bullet or numbering. This is the content of the header. I feel like we don't need to intend this

File: sending-a-message-to-a-service/README.md (hunk 3 above)
Comment (dulajdilshan) [reply sending-a-message-to-a-service/README.md]: Reason: This content belongs to # 2.

File: service-orchestration/README.md (hunk 2 above)
Comment (mindula) [reply service-orchestration/README.md:574]: Yes my bad

File: sending-a-message-to-a-service/README.md (hunk 3 above)
Comment (MaryamZi) [reply sending-a-message-to-a-service/README.md]: Updated, please check.

**Review by mindula — APPROVED** [review]

**Review by dulajdilshan — APPROVED** [review]

**Pull request merged**